comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
While I'm not sure that having it always print the message would be bad in this particular case (because I strongly suspect that any COM object trying to overwrite a Python keyword that isn't `True` or `False` will cause problems), I agree that that wasn't my intention.

Since I haven't figured out how to use `logging` yet, I was following a pattern I saw in another file, however I skipped over the `__verbose__ = __debug__` step at the top of the file.

What would you recommend I do here?